README.md

```diff
@@ -27,33 +27,44 @@ GRChombo is licensed under the BSD 3-Clause License. Please see LICENSE for deta
 
 ## Fawcett Tutorial
 
-First we need to get a copy of both the Chombo and GRChombo repositories.
+The following tutorial has been created to be aimed at Masters/early PhD
+students with some HPC experience. We also assume that you have an account set
+up on fawcett, and that you are happy with using GitHub. Inevitably it will not be 
+possible to cover everything, but if anything is unclear please do get in contact.
+
+To run simulations with GRChombo, first we need to get a copy of both the
+Chombo and GRChombo repositories.
 
 ### Getting Chombo and GRChombo
 
-Our copy of the Chombo repository can be found
-[here](https://github.com/GRChombo/Chombo). To get a copy, clone it using a
-command such as
+The GRChombo copy of the Chombo repository can be found
+[here](https://github.com/GRChombo/Chombo). To get a copy on your local
+machine, clone it using a command such as
```

Comment:
I don't think we want to talk about local machines as this tutorial is specific to running on fawcett. I think we should leave the line as I wrote it originally.
```suggestion
[here](https://github.com/GRChombo/Chombo). To get a copy, clone it using a
command such as
```